tests/Dom/ElementTest.php

```diff
@@ -169,8 +169,62 @@ public function getAddAmpActionData()
      */
     public function testAddAmpAction(Element $element, $event, $action, $expected)
```

Comment:
@schlessera  $event, $action and $expected params are unused now. Do we need to consider refactoring the `getAddAmpActionData` dataProvider method?

Reply:
Ah, yes, good catch. I forgot to remove the old code. Resolved via https://github.com/ampproject/amp-toolbox-php/pull/328/commits/2d48728317c93e9412b96baf2b51c65fabaeefe3